user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/Tag.java

```diff
@@ -8,4 +8,9 @@ public Tag(String text) {
     super(text);
     addClassName("tag");
   }
+
+  public Tag(String text, String toolTip) {
+    this(text);
+    setTitle(toolTip);
```

Comment:
What is the difference/ advantage of Tooltip.forComponent from vaadin? Is setTitle reliable?

Reply:
I was not aware of the Tooltip.forComponent method. Shall I change it?